packages/webdriver/src/index.js

```diff
@@ -50,24 +73,31 @@ export default class WebDriver {
         return monad(options.sessionId, commandWrapper)
     }
 
+    /* istanbul ignore next */
```

Comment:
While these are getters, I don't like ignoring all of these. This means that if someone does make a change to any of these getters in the future:
1.) we won't have tests
2.) the code coverage requirement won't be enforced if someone does expand these

Reply:
@abjerstedt added tests for the interface